WordPress/Classes/ViewRelated/Gutenberg/GutenbergNetworking.swift

```diff
@@ -0,0 +1,62 @@
+import Alamofire
+
+struct GutenbergNetworkRequest {
+    typealias Response = (Swift.Result<Any, NSError>) -> Void
+
+    private let path: String
+    private unowned let blog: Blog
+
+    init(path: String, blog: Blog) {
+        self.path = path
+        self.blog = blog
+    }
+
+    func request(response: @escaping Response) {
```

Comment:
I found quite confusing the `response`/`Response` naming for a callback, when the argument to that callback will be the actual response. Maybe we can use the more generic `completion`?

Reply:
sounds good! 👍